PlantDiary2022FS001/Pages/Index.cshtml.cs

```diff
@@ -40,6 +46,11 @@ private async Task<List<Specimen>> GetData()
             {
                 // get plant specimens at this location.
                 var task = client.GetAsync("http://plantplaces.com/perl/mobile/specimenlocations.pl?Lat=39.14455&Lng=-84.50939&Range=0.5&Source=location");
+                Task<HttpResponseMessage> plantTask = client.GetAsync("http://plantplaces.com/perl/mobile/viewplantsjsonarray.pl?WetTolerant=on");
+
+                String weatherEndpoint = "https://api.weatherbit.io/v2.0/current?&city=Cincinnati&country=USA&key=" + apiKey;
+                Task<HttpResponseMessage> weatherTask = client.GetAsync(weatherEndpoint);
+
```

Comment:
Move Asynchronous calls to the top of the method, so that they can start working as the method is processing.

Reply:
Ah!  yes.  This will speed up the program, as there will be less waiting.